docs: update UG (#177)
AY2122S2-CS2103T-T17-2/tp: docs/UserGuide.md

![Ui](images/Ui.png)

5. Type the command where prompted "Enter command here" and press Enter to execute it.
    * Example: typing **`help`** and pressing Enter will open the help window which would bring provide you with a link and direct you to the [Command Summary](#command-summary).

6. Refer to the [Features](#features) below for details of each command.

---

## Command Summary
### Understanding the command format
1. Almost all the details of the commands are needed to be given before the command can be executed properly.

>     * **Compulsory** details are denoted by angled-brackets such as \<mandatory details\>.
>     * **Optional** details of the command that are enclosed in square brackets such as [optional details].

Reviewer: can probs phrase it in the same format. For 
```suggestion
    * **Optional** details of the command are enclosed in square brackets such as [optional details].
```

        + Example: `command <recipe name or -x <recipe number from the list>> [optional details]`
        + A clearer example of this can be seen in the [delete command](#delete-an-existing-recipe-delete), or [edit command](#edit-an-existing-recipe-edit).
      

> 2. What does it mean when the input command says "... separated by pipe"?

Reviewer: instead of the word pipe, should it be just says"... seperated by |"? 
The next sentence will explain it

    * Pipe is typically located above the "Enter" button on the keyboard and can be typed by using `Shift` + ``\ ``, resulting in `|`.

>     * This means that the data you enter would be separated by the pipe symbol and would be as though you had entered them separately.

Reviewer: Sounds abit off.. we can say smt like pipe is like an OR or AND function....?

| [view a specific recipe](#view-an-existing-recipe-view)                                  | `view <recipe name>` <br></br> or <br></br> `view <recipe number from list>`                                                                                                                                                                                                                                                                                                                                                                                                                         |

[<sub>return to table of content</sub>](#table-of-content)

---

## Features

In this section we would be going over what each feature does, how to go about doing so, and what it achieves. <br>

<div markdown="block" class="alert alert-info">

> :information_source: :computer: **Quick note on [Save](#save-recipes) and [Load](#load-recipes)** these commands are done automatically, just like how Google documents automatically save your changes without you needing to explicitly run them. There are some advanced things that you can do with those features, and we recommend you jump to their section to find out more about them. <br><br>**Hint**: There's a way that you can share your recipe book with your friends if they also use McKitchen.

Reviewer: ```suggestion
:information_source: :computer: **Quick note on [Save](#save-recipes) and [Load](#load-recipes)** these commands are done automatically, just like how Google documents automatically save your changes without you needing to explicitly run them. There are some advanced things that you can do with those features, and we recommend you to explore these features to find out more. <br><br>**Hint**: If your friends is using McKitchen, there's a way for you to share your recipe book with them.
```

[<sub>return to table of content</sub>](#table-of-content)

### Reset to the default recipes: `reset`
Resets the existing recipes in McKitchen to the default ones that it initially came with.

Usage: `reset`

Example:

> - Oops! You've made the mistake of running `clear -f` before exploring the default recipes. Don't worry, we've got you. All you need to do is run:

Reviewer: Are there any other reasons to why we need to use "reset"?
